tests/tools/test_gcc.lua

```diff
@@ -991,10 +991,24 @@ end
 		test.contains({ }, gcc.getcflags(cfg))
 	end
 
+	function suite.cxxflags_onCpp2b()
+		cppdialect "C++2b"
+		prepare()
+		test.contains({ "-std=c++2b" }, gcc.getcxxflags(cfg))
+		test.contains({ }, gcc.getcflags(cfg))
```

Comment:
I know it is the same as others, but `test.contains({}, any_string)` would always be true, and can be removed.
`test.excludes({ "" -std_c++2b}, gcc.getcflags(cfg))` might make more sense though

Reply:
Nice catch @Jarod42, `test.contains({}, ...)` should trigger an error since it does nothing. I'll submit a PR that introduces an error for doing that, _and_ fix up any existing instances of that. I think we can merge this as-is though.